integ-test/src/test/java/org/opensearch/sql/sql/NestedIT.java

```diff
@@ -254,7 +254,7 @@ public void nested_function_and_field_with_order_by_clause() {
         rows("a", 4),
         rows("b", 2),
         rows("c", 3),
-        rows("zz", new JSONArray(List.of(3, 4))));
+        rows("zz", 3));
```

Comment:
Isn't a breaking change? Please, compare this with `nested` behavior in V1 and V2 @ 2.8.

Reply:
This is V1 behaviour, but my updates make this consistent with V2 behaviour. I believe this should be considered a bug and not a breaking change.